### PR TITLE
chore(react-chart): fix cutting of the last grid

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: packages/dx-react-chart/src/types/plugins.pane-layout.types.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/** @internal */
+export interface PaneLayoutProps {
```
Comment (DmitryBogomolov) [inline packages/dx-react-chart/src/types/plugins.pane-layout.types.ts]: Isn't is unused?

File: packages/dx-react-chart/src/plugins/pane-layout.tsx [hunk 2]
```diff
@@ -36,6 +39,8 @@ export class PaneLayout extends React.PureComponent {
             <TemplateConnector>
               {({ layouts }, { changeBBox }) => {
                 const { width, height } = layouts.pane;
+                const clipPathId = `clip_path_${numDefs}`;
```
Comment (DmitryBogomolov) [inline packages/dx-react-chart/src/plugins/pane-layout.tsx]: Unique clip path is bound to a component instance not to a particular template render occurrence.

File: packages/dx-react-chart/src/types/plugins.pane-layout.types.ts (hunk 1 above)
Comment (Krijovnick) [reply packages/dx-react-chart/src/types/plugins.pane-layout.types.ts]: yes, i'll remove it

File: packages/dx-react-chart/src/plugins/pane-layout.tsx (hunk 2 above)
Comment (DmitryBogomolov) [reply packages/dx-react-chart/src/plugins/pane-layout.tsx]: I meant - *should be* bound.

**Pull request merged**